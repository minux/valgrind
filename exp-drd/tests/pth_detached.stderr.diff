0a1
> exp-drd, a data race detector.
1a3,12
> Thread 3:
> Conflicting load by thread_func2[0] at 0x........ size 8
>    at 0x........: (within libpthread-?.?.so)
>    by 0x........: start_thread (in libpthread-?.?.so)
>    by 0x........: clone (in /...libc...)
> Allocation context: /lib64/libpthread-2.5.so, libpthread.so.0:Text
> Other segment start (thread_func1[0])
>    (thread finished, call stack no longer available)
> Other segment end (thread_func1[0])
>    (thread finished, call stack no longer available)
3c14
< ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
---
> ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
