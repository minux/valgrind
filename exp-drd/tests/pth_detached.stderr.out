exp-drd, a data race detector.

Thread 3:
Conflicting load by thread_func2[0] at 0x........ size 8
   at 0x........: (within libpthread-?.?.so)
   by 0x........: start_thread (in libpthread-?.?.so)
   by 0x........: clone (in /...libc...)
Allocation context: /lib64/libpthread-2.5.so, libpthread.so.0:Text
Other segment start (thread_func1[0])
   (thread finished, call stack no longer available)
Other segment end (thread_func1[0])
   (thread finished, call stack no longer available)

ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
